docs/integrations/store/configu/authorization.mdx

```diff
@@ -48,7 +48,7 @@ To assign roles to your [organization members](/integrations/store/configu/membe
 ## Creating a custom role
 
 Take control of your organization's access management with our flexible and dynamic custom roles feature.
-Every organization is unique, with specific requirements and workflows. That's why we empower you to create custom roles, allowing you to fine-tune access permissions according to your exact neeConstruction.
```

Comment:
is it related? or you just caught it along the way?
can you review all current configu platform documentation and let us know on mismatches?

Reply:
Something i caught along the way, and of course, I'll try and hunt for more mismatches.